vars/rollingUpgradePipeline.groovy

```diff
@@ -60,6 +60,9 @@ def call(Map pipelineParams) {
             string(defaultValue: "${pipelineParams.get('test_config', '')}",
                    description: 'Test configuration file',
                    name: 'test_config')
+            string(defaultValue: "${pipelineParams.get('base_versions', [])}",
```

Comment:
Also, there is should be problem with type conversion. It should be parsed from `String` to the `List`

Reply:
yes, i'm stuck on it right now.

Reply:
done, it now works